src/main/java/lottery/domain/LotteryRank.java

```diff
@@ -0,0 +1,35 @@
+package lottery.domain;
+
+import java.util.Arrays;
+
+public enum LotteryRank {
+    FIRST_PRIZE(6, 2_000_000_000),
+    SECOND_PRIZE(5, 1_500_000),
+    THIRD_PRIZE(4, 50_000),
+    FOURTH_PRIZE(3,5_000);
+
+    private final int matchNumberCounts;
+    private final long prizeMoney;
+
+    private LotteryRank(int matchNumberCounts, long prizeMoney) {
+        this.matchNumberCounts = matchNumberCounts;
+        this.prizeMoney = prizeMoney;
+    }
+
+    public static LotteryRank valueOf(int matchNumberCounts) {
+        return Arrays.stream(LotteryRank.values())
+                .filter(lotteryRank -> lotteryRank.getMatchNumberCounts() == matchNumberCounts)
+                .findAny()
+                .orElseThrow(() -> new IllegalArgumentException(ErrorMessages.INVALID_MATCH_NUMBER_COUNTS));
+    }
+
+    public double calculatePrizeMoneySum(int winnerTicketCounts) {
+        return (double)prizeMoney * winnerTicketCounts;
```

Comment:
```suggestion
        return (double) prizeMoney * winnerTicketCounts;
```
intellij 를 IDE 로 사용하실 경우 reformat 기능을 활용하시면 미리 정해놓은 규칙에 따라 컨벤션을 자동을 맞출 수 있습니다!

Reply:
brace와 space 관련은 IDE로 reformat했습니다. :)